docs/language_guide.md

```diff
@@ -522,37 +634,57 @@ struct Rectangle {
 ```
 
 ### Instances
-The `Rectangle` structure definition describes only what a `Rectangle` would look like and what functionality they contain. They don't describe specific rectangles. To do that you create an instance of that structure.
+
+The declaration of a struct only describes what types of variables it contains, what their initial values are, and what functions may be used to modify or access the struct data. To create concrete instances, each with individual data values, an instance has to be created, by calling the initialiser of a struct.
+
+```swift
+<struct-name>(<initialiser-parameter-values>)
+```
+
+Example:
 
 ```swift
 let someRectangle: Rectangle = Rectangle()
 ```
 
-When you create an instance, you initialize that structure with it's initial values - in this case a width and height of 0. This process can also be done manually using the `init` special declaration. You can access the properties of the current structure with the special keyword `self`
+When an instance is created, it is initialised with its initial values – in this case a width and heigth of `0`. This process can also be done manually using an [initialiser](#initialisers). You can access the properties of the current struct with the special keyword `self`.
```

Comment:
heigth => height

Reply:
Importantly an initializer HAS to be used if default value initialization is not used. @ codegen stage, initialized struct member declarations are pasted into the constructor function so they perform the same function